tests/DependencyInjection/AbstractDoctrineExtensionTestCase.php

```diff
@@ -80,6 +81,7 @@ final protected function loadMinimalOrmConfig(ContainerBuilder $container): void
         $loadPhp->import('minimal_orm.php');
     }
 
+    #[IgnoreDeprecations('Since symfony/dependency-injection 7.4: XML configuration format is deprecated, use YAML or PHP instead.')]
```

Comment:
Could we ignore deprecations only for the Xml Test Case (does this support to be added at the class level ?)